envbuilder.go

```diff
@@ -1063,3 +1062,20 @@ func findDevcontainerJSON(options Options) (string, string, error) {
 
 	return "", "", errors.New("can't find devcontainer.json, is it a correct spec?")
 }
+
+// maybeDeleteFilesystem wraps util.DeleteFilesystem with a guard to hopefully stop
+// folks from unwittingly deleting their entire root directory.
+func maybeDeleteFilesystem(force bool) error {
+	kanikoDir, ok := os.LookupEnv("KANIKO_DIR")
+	if !ok || strings.TrimSpace(kanikoDir) != MagicDir {
+		if force {
+			_, _ = fmt.Fprintf(os.Stderr, "WARNING! BYPASSING SAFETY CHECK! THIS WILL DELETE %s!\n", kanikoDir)
```

Comment:
Idea: how about `time.Sleep(2*time.Second)`, "you have to seconds to abort"?

Reply:
lol I was actually thinking about doing this. I'd go for ten seconds just to be sure.